wormhole-connect/src/AppRouter.tsx

```diff
@@ -102,9 +102,15 @@ function AppRouter(props: Props) {
     return props.config?.useRedesign ? <BridgeV2 /> : <Bridge />;
   }, [props.config?.useRedesign]);
 
+  const walletSelector = useMemo(() => {
+    return props.config?.useRedesign || !showWalletModal ? null : (
```

Comment:
Adding a check for the useRedesign flag not to render the old wallet modal when it's set.